### ISSUE TITLE
How to convert blinear lstm model into tftrt

### ISSUE THREAD

**Issue by Csedge** [tensorflow/tensorrt#186]
i have a model that base on the tensorflow. That is a blinear lstm. When I try to frozen the graph according to the tftrt guide line .the data type was convert in wrong . It automatically from tensor to operator. So it convert failed.

**Comment by tfeher** [comment]
TF-TRT currently does not convert the LSTM op (see [supported ops](https://docs.nvidia.com/deeplearning/frameworks/tf-trt-user-guide/index.html#supported-ops)). 

TensorRT itself supports LSTMs through its [RNN layer](https://docs.nvidia.com/deeplearning/sdk/tensorrt-api/python_api/infer/Graph/Layers.html#irnnv2layer), but TF-TRT does not convert the TF LSTM into TensorRT LSTM. You could try using TensorRT directly if it is an option for you.

**Comment by agrija9** [comment]
@tfeher do you know if TF-TRT now supports LSTM layers?

**Comment by tfeher** [comment]
No, we still do not have support for LSTM in TF-TRT. Alternative to TF-TRT, you could try if the [tf2onnx -> onnx-tensorrt route](https://developer.nvidia.com/blog/speeding-up-deep-learning-inference-using-tensorflow-onnx-and-tensorrt/) works for you.

**Comment by agrija9** [comment]
I will try that route then, thanks for your reply!

One more question, have you encountered examples of TensorRT for time series forecasting using LSTM? Or is TensorRT mostly computer vision oriented?
